deploy/acm/crds/action.open-cluster-management.io_managedclusteractions.yaml

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
Is this included just for testing? How does the operator ensure this CRD is available when installed via subscription?

Reply:
This is just for testing. If ACM is installed then the CRD should exist. I didn't have to do anything extra on my environment to get these CRDs.